book/website/communication/citable/citable-linking.md

```diff
@@ -62,6 +66,440 @@ Whichever route you decide to take, it is important to double-check the proof of
+
+**After deposit:**
+1. Contact the repository to update metadata
+2. Provide complete funding information:
+   - Funder name (and Funder ID if you know it)
+   - Grant number
+   - Grant title
+
+**Repository-specific guidance:**
+
+**Zenodo:**
+- "Funding" section during upload
+- Type-ahead search of Funder Registry
+- Add multiple funders
+- Fields for grant number and details
+
+**Figshare:**
+- "Funding" field in metadata editor
+- Free text, but try to match official grant information
+
+**Dryad:**
+- "Funding information" during submission
+- Connected to journal submission data when available
+
+**OSF:**
+- Not currently supported in core OSF metadata
+- Can include in project description/wiki
+- Supported in some OSF-integrated services
+
+### Best Practices for Funding Citation
+
+- **Add funding to all outputs**: Not just papers - include in data, code, protocols, and other outputs
+- **Use official grant numbers**: Match the funder's format exactly
+- **Include all funders**: If multiple organizations contributed, cite them all
+- **Add early**: Include funding information when you first deposit outputs
+- **Be consistent**: Use the same grant number format across all outputs from that grant
+- **Check with your funder**: Some funders have specific requirements for how grants should be cited
+
+### Example Funding Metadata
+
+A dataset might include:
+```
+Funder: UK Research and Innovation
+Funder Identifier: https://ror.org/001aqnf71
+Award Number: MR/V012345/1
+Award Title: Understanding Climate Change Impacts on Grassland Ecosystems
+
+Funder: Natural Environment Research Council
+Funder Identifier: https://ror.org/02b5d8509
+Award Number: NE/X067891/1
+```
+
+Note: A project might list both the parent organization (UKRI) and the specific council (NERC) if both provided funding or if the specific council should be credited.
+
+(cm-citable-linking-people)=
+## People and Organizations in Metadata
+
+Just as research outputs have persistent identifiers, so do people and organizations.
+Including these identifiers in your research output metadata creates a rich network of connections.
+
+### ORCID: Persistent Identifiers for Researchers
+
+ORCID (Open Researcher and Contributor ID) provides unique identifiers for researchers that distinguish them from everyone else, even people with identical names.
+
+For comprehensive guidance on ORCID, see our {ref}`dedicated ORCID chapter<cm-citable-orcid>`.
+
+**Key points for connection metadata:**
+- Include your ORCID iD when depositing research outputs
+- Add ORCID iDs for all co-authors when possible
+- The ORCID iD connects all your outputs across different repositories and publications
+- Your ORCID profile can automatically collect works that cite your ORCID iD
+
+**Format:** ORCIDs are 16-digit numbers formatted as `0000-0001-2345-6789`
+As URLs: `https://orcid.org/0000-0001-2345-6789`
+
+### ROR: Persistent Identifiers for Organizations
```

Comment:
This feels out of place here, a explanation/definition of ROR should probably sit alongside the same for ORCID.